lib/osf-components/addon/components/banners/view-only-link/template.hbs

```diff
@@ -13,9 +13,10 @@
             data-test-view-normally
             data-analytics-name='Stop view-only link'
             @type='default'
-            @onClick={{action this.stopViewOnly}}
+            {{on 'click' this.stopViewOnly}}
+            {{on 'click' @manager.cancel}}
```

Comment:
Where's the `@manager` coming from and what manager is this here?

Reply:
@manager was added to be locally tested. this should not made it into the PR and has now been removed.